### PR TITLE
remove @see to super methods

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

**Comment by grimreaper** [comment]
tabs and spaces fixed

**Comment by thpr** [comment]
pls resolve conflicts

File: code/src/test/pcgen/io/PCGFileTest.java [hunk 1]
```diff
@@ -11,7 +11,7 @@
 {
 
 	/**
-	 * Constructs a new {@link PCGFileTest}.
+	 * Constructs a new .
```
Comment (LegacyKing) [inline code/src/test/pcgen/io/PCGFileTest.java]: Uh, we leave floating text here?

File: code/src/test/pcgen/io/PCGFileTest.java [hunk 2]
```diff
@@ -21,7 +21,7 @@ public PCGFileTest()
 	}
 
 	/**
-	 * Constructs a new {@link PCGFileTest} with the given <var>name</var>.
+	 * Constructs a new  with the given <var>name</var>.
```
Comment (LegacyKing) [inline code/src/test/pcgen/io/PCGFileTest.java]: Why?

File: code/src/test/pcgen/PCGenTestCase.java [hunk 3]
```diff
@@ -84,7 +84,7 @@ public PCGenTestCase(final String name)
 	}
 	
 	/**
-	 * Fixes {@link TestCase#runBare()} to not swallow a throwable from {@link
+	 * Fixes  to not swallow a throwable from {@link
```
Comment (LegacyKing) [inline code/src/test/pcgen/PCGenTestCase.java]: Why?

**Comment by grimreaper** [comment]
this was reworked in a different branch

**Pull request closed without merging**